src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.xaml.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.LocalServices;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Geoprocessing;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace ArcGISRuntime.WPF.Samples.LocalServerGeoprocessing
+{
+    public partial class LocalServerGeoprocessing
+    {
+        // Hold a reference to the local geoprocessing service
+        private LocalGeoprocessingService _gpService;
+
+        // Hold a reference to the task
+        private GeoprocessingTask _gpTask;
+
+        // Hold a reference to the job
+        private GeoprocessingJob _gpJob;
+
+        public LocalServerGeoprocessing()
+        {
+            InitializeComponent();
+
+            // set up the sample
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create a map and add it to the view
+            MyMapView.Map = new Map(Basemap.CreateLightGrayCanvasVector());
+
+            // Load the tiled layer
```

Comment:
Perhaps a bit more triple slash comments for the next few lines of code:
 47   +            // Get the file/path to the raster file from the DataManager helper functions  
 48   +            string rasterPath = await GetRasterPath();  
 49   +  
                   // Create a new TileCache from the raster file path
 50   +            TileCache myTileCache = new TileCache(rasterPath);  
 51   +  
                   // Create an ArcGISTiledLater from the TileCache
 52   +            ArcGISTiledLayer tiledLayer = new ArcGISTiledLayer(myTileCache);